freva-rest/src/databrowser_api/core.py

```diff
@@ -753,6 +764,28 @@ async def init_stream(self) -> Tuple[int, SearchResult]:
             primary_facets=[],
         )
 
+    def _join_facet_queries(
+        self, key: str, facets: List[str]
+    ) -> Tuple[str, str]:
+        """Create lucene search contain and NOT contain search queries"""
+
+        negative, positive = [], []
+        for search_value in facets:
+            if key not in self.uniq_keys:
+                search_value = search_value.lower()
```

Comment:
what happens with this case, is it lost? Could this be:
```python
                positive.append(search_value.lower())
```

instead?

Reply:
We don't need that. because `search_value` is already transformed to lowercase when the search key is not file or uri. We do that because files/uri stored without making the lowercase (we need to access them)